can/io/sqlite.py

```diff
@@ -64,6 +67,7 @@ def close(self):
 
 
 # Backward compatibility
+# TODO remove in later releases?
```

Comment:
Yeah we can probably add a future deprecation warning for a release or two.

Reply:
Uhm, any ideas on how to do that?

Reply:
One way is to use the [deprecated](https://github.com/tantale/deprecated) library